MIT.md

```diff
@@ -1,5 +1,5 @@
-Copyright 2022, Youta Christabelle Please delete this line and the next one APP TYPE can be a webpage/website, a web app, a software and so on Permission is hereby granted, free of charge, to any person obtaining a copy of this web app and associated documentation files, to deal in the web app without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the web app, and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+Copyright 2021, [Youta Christabelle] Please delete this line and the next one APP TYPE can be a webpage/website, a web app, a software and so on Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
```

Comment:
- [ ] Kindly customize your license file, remove instructions on how to write your license, also add your web application name instead of using `  [APP TYPE]`.